src/routes/post.tsx

```diff
@@ -61,7 +67,7 @@ export function Post(): ReactElement | ReactElement[] | null {
           <Text>
             <strong>User Id:</strong> {post.data.userId}
           </Text>
-          {comments.data.map((comment) => (
+          {comments.data.map((comment: PostComment) => (
```

Comment:
typescript will automatically infer the type of `comment`, you don't need to explicitly set it